Misc/NEWS.d/next/Library/2018-02-26-09-08-07.bpo-32257.6ElnUt.rst

```diff
@@ -0,0 +1,2 @@
+The ssl module now contains OP_NO_RENEGOTIATION constant, available with
+OpenSSL 1.1.0n or 1.1.1.
```

Comment:
Your PR description says 1.1.0h, but this says 1.1.0n. Which is it?

Reply:
D'oh, it's a typo. Fixed